javascript/javascript1/week3/travel.js

```diff
@@ -0,0 +1,15 @@
+const travelInformation = { speed: 50, destinationDistance: 432 };
+
+function myTravelTime(myInformation) {
+  let time = myInformation.destinationDistance / myInformation.speed;
+  decimalTime = time * 60 * 60;
```

Comment:
- Well done 👌🏼 just some notes
- 'decimalTime' is not defined. You should declare it 
```js
 let decimalTime = time * 60 * 60;
```
- the other variables can be `const` because they are never reassigned